app/frontend/src/cljs/teet/dashboard/dashboard_view.cljs

```diff
@@ -130,11 +140,17 @@
              (doall
                (for [{:notification/keys [type]
                       :meta/keys [created-at]
+                      task-part-name :notification/target
                       id :db/id} notifications]
                  ^{:key (str id)}
                  [:div
                   [buttons/link-button {:on-click #(e! (notification-controller/->NavigateTo id))}
-                   (tr-enum type) " " (fmt/date-time created-at)]])))]
+                   (if (is-task-part-notification? type)    ;; In case of a task part notification, the translated message includes a task part name and number
```

Comment:
related to above, why are task part notifications special case? feels like there should be general approach... but perhaps this can be left as refactoring for the future

Reply:
It needed a change as task part notifications have the task part name in the title as well, i.e. "Task part #5 "Important Documents" was submitted for review". All other notifications just describe the action.